py3status/modules/nvidia_temp.py

```diff
@@ -32,9 +36,21 @@ class Py3status:
     """
     # configuration parameters
     cache_timeout = 10
-    format_prefix = "GPU: "
-    format_units = u"°C"
-    temp_separator = '|'
+    format = "GPU: {gpu_temp}"
+    gpu_temp = u"{temp}°C"
```

Comment:
I'd call this `format_temp` as it uses formatter and has placeholder. but default `'{temp}{temp_unit}'` and have the °C set by `temp_unit` like we are doing in #555 

for format I'd have the placeholder `{gpu_temps}` as there may be multiple ones.

we can create the output for `gpu_temps`